packages/libs/coreui/src/components/inputs/SelectList.tsx

```diff
@@ -29,12 +34,23 @@ export default function SelectList<T>({
   );
 
   const onClose = () => {
+    if (instantUpdate) return; // the next two effects take care of the following when constantly updating
+
     onChange(selected);
     setButtonDisplayContent(
       selected.length ? selected.join(', ') : defaultButtonDisplayContent
     );
   };
 
+  /**
+   * Keep caller up to date with any selection changes
+   */
+  useEffect(() => {
+    if (instantUpdate) {
+      onChange(selected);
+    }
+  }, [onChange, selected, instantUpdate]);
```

Comment:
This should be called in a callback used as the `onChange` prop of `CheckboxList`. We don't currently have one defined, so it will be a new callback.